build fails on lodash.groupBy

node: v6.10.2
yarn: 0.22.0

The command `npm run build` fails with the following error:

```
ERROR in /home/yfful/documents/code/vue-devtools/~/buble-loader?{"target":null,"objectAssign":"Object.assign"}!/home/yfful/documents/code/vue-devtools/~/vue-loader/lib/selector.js?type=script&index=0!/home/yfful/documents/code/vue-devtools/src/devtools/views/components/ComponentInspector.vue
Module not found: Error: Can't resolve 'lodash.groupBy' in '/home/yfful/documents/code/vue-devtools/src/devtools/views/components'
 @ /home/yfful/documents/code/vue-devtools/~/buble-loader?{"target":null,"objectAssign":"Object.assign"}!/home/yfful/documents/code/vue-devtools/~/vue-loader/lib/selector.js?type=script&index=0!/home/yfful/documents/code/vue-devtools/src/devtools/views/components/ComponentInspector.vue 37:0-36
 @ /home/yfful/documents/code/vue-devtools/src/devtools/views/components/ComponentInspector.vue
 @ /home/yfful/documents/code/vue-devtools/~/buble-loader?{"target":null,"objectAssign":"Object.assign"}!/home/yfful/documents/code/vue-devtools/~/vue-loader/lib/selector.js?type=script&index=0!/home/yfful/documents/code/vue-devtools/src/devtools/views/components/ComponentsTab.vue
 @ /home/yfful/documents/code/vue-devtools/src/devtools/views/components/ComponentsTab.vue
 @ /home/yfful/documents/code/vue-devtools/~/buble-loader?{"target":null,"objectAssign":"Object.assign"}!/home/yfful/documents/code/vue-devtools/~/vue-loader/lib/selector.js?type=script&index=0!/home/yfful/documents/code/vue-devtools/src/devtools/App.vue
 @ /home/yfful/documents/code/vue-devtools/src/devtools/App.vue
 @ /home/yfful/documents/code/vue-devtools/src/devtools/index.js
 @ ./src/devtools.js
```

--- a/src/devtools/views/components/ComponentInspector.vue
+++ b/src/devtools/views/components/ComponentInspector.vue
@@ -34,7 +34,7 @@ import ScrollPane from 'components/ScrollPane.vue'
 import ActionHeader from 'components/ActionHeader.vue'
 import StateInspector from 'components/StateInspector.vue'
 import { searchDeepInObject } from 'src/util'
-import groupBy from 'lodash.groupBy'
+import groupBy from 'lodash.groupby'
 
 const isChrome = typeof chrome !== 'undefined' && chrome.devtools
 

